easybuild/easyconfigs/p/PythonPlus/PythonPlus-3.7.4-foss-2018b-v20.11.1.eb

```diff
@@ -0,0 +1,792 @@
+        'source_urls': ['https://files.pythonhosted.org/packages/source/p/prometheus_client'],
+        'checksums': ['71cd24a2b3eb335cb800c7159f423df1bd4dcd5171b234be15e3f31ec9f622da'],
+    }),
+    ('nose_warnings_filters', '0.1.5', {
+        'source_urls': ['https://files.pythonhosted.org/packages/source/n/nose_warnings_filters'],
+        'checksums': ['456c5b2ccca24e1d00a7b558274ebf9318305813dcb9585951a73ae11d76bb9d'],
+    }),
+    ('nose-exclude', '0.5.0', {
+        'source_urls': ['https://files.pythonhosted.org/packages/source/n/nose-exclude'],
+        'checksums': ['f78fa8b41eeb815f0486414f710f1eea0949e346cfb11d59ba6295ed69e84304'],
+    }),
+    ('selenium', '3.141.0', {
+        'source_urls': ['https://files.pythonhosted.org/packages/source/s/selenium'],
+        'checksums': ['deaf32b60ad91a4611b98d8002757f29e6f2c2d5fcaf202e1c9ad06d6772300d'],
+    }),
+    ('ipython', '7.7.0', {
+        'modulename': 'IPython',
+        'source_urls': ['https://files.pythonhosted.org/packages/source/i/ipython'],
+        'checksums': ['1d3a1692921e932751bc1a1f7bb96dc38671eeefdc66ed33ee4cbc57e92a410e'],
+    }),
+    ('scikit-bio', '0.5.5', {
+        'modulename': 'skbio',
+        'source_urls': ['https://files.pythonhosted.org/packages/source/s/scikit-bio'],
+        'checksums': ['9fa813be66e88a994f7b7a68b8ba2216e205c525caa8585386ebdeebed6428df'],
+    }),
+    ('biopython', '1.74', {
+        'modulename': 'Bio',
+        'source_urls': ['https://files.pythonhosted.org/packages/source/b/biopython'],
+        'checksums': ['25152c1be2c9205bf80901fc49adf2c2efff49f0dddbcf6e6b2ce31dfa6590c0'],
+    }),
+    ('ete3', '3.1.1', {
+        'source_urls': ['https://files.pythonhosted.org/packages/source/e/ete3'],
+        'checksums': ['870a3d4b496a36fbda4b13c7c6b9dfa7638384539ae93551ec7acb377fb9c385'],
+    }),
+    ('click', '7.0', {
+        'source_tmpl': 'Click-%(version)s.tar.gz',
+        'source_urls': ['https://files.pythonhosted.org/packages/source/c/click'],
+        'checksums': ['5b94b49521f6456670fdb30cd82a4eca9412788a93fa6dd6df72c94d5a8ff2d7'],
+    }),
+    ('livereload', '2.6.1', {
+        'source_urls': ['https://files.pythonhosted.org/packages/source/l/livereload'],
+        'checksums': ['89254f78d7529d7ea0a3417d224c34287ebfe266b05e67e51facaf82c27f0f66'],
+    }),
+    ('PyYAML', '5.1.2', {
+        'modulename': 'yaml',
+        'source_urls': ['https://files.pythonhosted.org/packages/source/P/PyYAML'],
+        'checksums': ['01adf0b6c6f61bd11af6e10ca52b7d4057dd0be0343eb9283c878cf3af56aee4'],
+    }),
+    ('Markdown', '3.1.1', {
+        'source_urls': ['https://files.pythonhosted.org/packages/source/M/Markdown'],
+        'checksums': ['2e50876bcdd74517e7b71f3e7a76102050edec255b3983403f1a63e7c8a41e7a'],
+    }),
+    ('mkdocs', '1.0.4', {
+        'source_urls': ['https://files.pythonhosted.org/packages/source/m/mkdocs'],
+        'checksums': ['17d34329aad75d5de604b9ed4e31df3a4d235afefdc46ce7b1964fddb2e1e939'],
+    }),
+
```

Comment:
Er is hier een loze witregel in geslopen. Dat suggereert wat handmatige hacks.... maar zou geen kwaad moeten kunnen...